reservation.yaml.sample

```diff
@@ -127,10 +127,18 @@ registry:
     - ceph2.rennes.grid5000.fr
 
 # ############################################### #
-# Monitoring stack parameters                     #
+# Enos Customizations                             #
 # ############################################### #
 enable_monitoring: no
 
+# Patch kolla-ansible source
```

Comment:
Actually the patch is neither enabled nor available. I add this to give an usage example of the new `{{ cwd }}` key. Do you have a better idea to show the usage of that key?